common/frame_session.go

```diff
@@ -96,6 +103,11 @@ func NewFrameSession(ctx context.Context, session *Session, page *Page, parent *
 		eventCh:              make(chan Event),
 		childSessions:        make(map[cdp.FrameID]*FrameSession),
 		logger:               logger,
+		serializer: &logrus.Logger{
+			Out:       state.Logger.Out,
+			Level:     state.Logger.Level,
+			Formatter: &consoleLogFormatter{state.Logger.Formatter},
+		},
```

Comment:
```suggestion
		serializer: &logrus.Logger{
			Out:       logger.log.Out,
			Level:     logger.log.Level,
			Formatter: &consoleLogFormatter{logger.log.Formatter},
		},
```

Reply:
Sure, resolved by 152b56d.

Reply:
Nice.
Meta nitpick: You can click on the `Commit Suggestion` button to apply the suggestion here. You can even send multiple suggestions in a single commit in the files tab.

Reply:
Yeah, I know, I was just editing the file manually anyway :wink: